File: trunk/yyandroid/src/com/yy/android/vo/Project.java
package com.yy.android.vo;

import java.io.Serializable;
import java.util.Date;

public class Project implements Serializable{
	private static final long serialVersionUID = 1L;
	private int F_ID;
	private String F_Name;
	private Date F_StartDate;
	private Date F_EndDate;
	private String F_Description;
	private boolean F_IsDelete;
	private int F_OperatorID;
	private Date F_OperateTime = new Date();

	public int getF_ID() {
		return F_ID;
	}

	public void setF_ID(int in_F_ID) {
		F_ID = in_F_ID;
	}

	public String getF_Name() {
		return F_Name;
	}

	public void setF_Name(String in_F_Name) {
		F_Name = in_F_Name;
	}

	public Date getF_StartDate() {
		return F_StartDate;
	}

	public void setF_StartDate(Date in_F_StartDate) {
		F_StartDate = in_F_StartDate;
	}

	public Date getF_EndDate() {
		return F_EndDate;
	}

	public void setF_EndDate(Date in_F_EndDate) {
		F_EndDate = in_F_EndDate;
	}

	public String getF_Description() {
		return F_Description;
	}

	public void setF_Description(String in_F_Description) {
		F_Description = in_F_Description;
	}

	public boolean isF_IsDelete() {
		return F_IsDelete;
	}

	public void setF_IsDelete(boolean in_F_IsDelete) {
		F_IsDelete = in_F_IsDelete;
	}

	public int getF_OperatorID() {
		return F_OperatorID;
	}

	public void setF_OperatorID(int in_F_OperatorID) {
		F_OperatorID = in_F_OperatorID;
	}

	public Date getF_OperateTime() {
		return F_OperateTime;
	}

	public void setF_OperateTime(Date in_F_OperateTime) {
		F_OperateTime = in_F_OperateTime;
	}

}

File: trunk/CSClient/Library/Library.Model/Entriy/Project.cs
﻿/**  版本信息模板在安装目录下，可自行修改。
* Project.cs
*
* 功 能： N/A
* 类 名： Project
*
* Ver    变更日期             负责人  变更内容
* ───────────────────────────────────
* V0.01  2015/6/24 16:29:45   N/A    初版
*
* Copyright (c) 2012 Maticsoft Corporation. All rights reserved.
*┌──────────────────────────────────┐
*│　此技术信息为本公司机密信息，未经本公司书面同意禁止向第三方披露．　│
*│　版权所有：动软卓越（北京）科技有限公司　　　　　　　　　　　　　　│
*└──────────────────────────────────┘
*/
using System;
namespace Library.Model
{
	/// <summary>
	/// Project:实体类(属性说明自动提取数据库字段的描述信息)
	/// </summary>
	[Serializable]
	public partial class Project
	{
		public Project()
		{}
		#region Model
		private int _f_id;
		private string _f_name;
		private DateTime? _f_startdate;
		private DateTime? _f_enddate;
		private string _f_description;
		private bool _f_isdelete= false;
		private int _f_operatorid;
		private DateTime _f_operatetime;
		/// <summary>
		/// 
		/// </summary>
		public int F_ID
		{
			set{ _f_id=value;}
			get{return _f_id;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_Name
		{
			set{ _f_name=value;}
			get{return _f_name;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? F_StartDate
		{
			set{ _f_startdate=value;}
			get{return _f_startdate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime? F_EndDate
		{
			set{ _f_enddate=value;}
			get{return _f_enddate;}
		}
		/// <summary>
		/// 
		/// </summary>
		public string F_Description
		{
			set{ _f_description=value;}
			get{return _f_description;}
		}
		/// <summary>
		/// 
		/// </summary>
		public bool F_IsDelete
		{
			set{ _f_isdelete=value;}
			get{return _f_isdelete;}
		}
		/// <summary>
		/// 
		/// </summary>
		public int F_OperatorID
		{
			set{ _f_operatorid=value;}
			get{return _f_operatorid;}
		}
		/// <summary>
		/// 
		/// </summary>
		public DateTime F_OperateTime
		{
			set{ _f_operatetime=value;}
			get{return _f_operatetime;}
		}
		#endregion Model

	}
}

